src/Stack/Options/DotParser.hs

```diff
@@ -102,4 +105,5 @@ listDepsOptsParser = subparser
                      (  formatSubCommand "text" "Print dependencies as text (default)" listDepsTextParser
                      <> formatSubCommand "tree" "Print dependencies as tree" listDepsTreeParser
                      <> formatSubCommand "json" "Print dependencies as JSON" listDepsJsonParser
+                     <> formatSubCommand "constraints" "Print exact cabal constraints" listDepsConstraintsParser
```

Comment:
Cabal is now referred to as 'Cabal', rather than 'cabal', consistently in Stack documentation.

I suggest this read 'Print dependencies as Cabal constraints`, to follow the pattern of the other subcommands.

I would place the new subcommand immediately after `text`, given the similarity between 'text' and the new subcommand.

I suggest that the new subcommand is named `cabal`, rather than `constraints` -  it is shorter (always good) and it more directly describes its intended use - that is, with Cabal (the tool).

Reply:
All done.